Task1_KFox/app/index.html

```diff
@@ -7,142 +7,187 @@
         <link rel="stylesheet" href="css/styles.css">
     </head>
     <body>
-        <header></header>
-        <div id="landingPage"></div>
-        <div id="recentlyPlayed">
-            <h1>Recently Played</h1>
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">George Thorogood &amp; Destroyers</div>
-                <div class="recentlyPlayed-title">Bad to the Bone</div>
-                <div class="recentlyPlayed-length">9:36</div>
-            </div>
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">Lynyrd Skynyrd</div>
-                <div class="recentlyPlayed-title">Free Bird</div>
-                <div class="recentlyPlayed-length">3:31</div>
-            </div> 
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">Led Zeppelin</div>
-                <div class="recentlyPlayed-title">Stairway to Heaven</div>
-                <div class="recentlyPlayed-length">5:36</div>
-            </div>            
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">The Who</div>
-                <div class="recentlyPlayed-title">Won't Get Fooled Again</div>
-                <div class="recentlyPlayed-length">4:36</div>
-            </div>            
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">The Rolling Stones</div>
-                <div class="recentlyPlayed-title">You Can't Always Get What You Want</div>
-                <div class="recentlyPlayed-length">9:16</div>
-            </div>            
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">Guns N'Roses</div>
-                <div class="recentlyPlayed-title">Patience</div>
-                <div class="recentlyPlayed-length">6:11</div>
-            </div>            
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">Cinderella</div>
-                <div class="recentlyPlayed-title">Don't Know What You Got (Till It's Gone)</div>
-                <div class="recentlyPlayed-length">4:39</div>
+        <div id="landingPage">
+            <header>                
```

Comment:
Since there could be multiple number of headers at the page, you should specify class or id here.